### DISCUSSION TITLE
If my email template contains Korean characters, I can't save it?

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [supabase/supabase#30384] [Questions] [answered]
## Korean characters

<img width="647" alt="화면 캡처 2024-11-09 101534" src="https://github.com/user-attachments/assets/66430fb1-7066-4233-837e-21a8410b664d">

## No korean characters

<img width="645" alt="화면 캡처 2024-11-09 101634" src="https://github.com/user-attachments/assets/ccad3045-5165-472d-9d47-8ffce0db6d41">

**Comment by GaryAustin1** [comment, marked as answer]
Are you using your SMTP provider.  I've been told that the Spam Assassin warnings stop the default Supabase provider from working, but should just generate a warning if you are using your own provider.

Otherwise I'd file an issue or contact support as this is a new feature as of last week.

**Reply by ryuhangyeong** [reply]
I'm using `resend`.

Can I leave this as an issue?

**Reply by GaryAustin1** [reply]
I would file an issue in supabase/supabase issues.

**Reply by ryuhangyeong** [reply]
I'm not good at English.

Nonetheless, I am sharing my current situation in the hope that it may be helpful to you.

## After initial project setup

- Even if the email template contains Korean, it can be saved

## Existing project

- If the email template contains Korean, it cannot be modified.

**Reply by joshenlim** [reply]
hey there @ryuhangyeong! I'm able to verify on prod that the templates should be able to be saved irregardless of the spam warnings if you've set up custom SMTP on your Supabase project.

Can I verify real quick if you've already done so in the [Auth settings](https://supabase.com/dashboard/project/_/settings/auth) page?
![image](https://github.com/user-attachments/assets/5cd95011-e7fd-4cc5-96a1-923312ba0cfe)

Your templates should look like this once you've got Custom SMTP set up - notice there's no longer the "Please rectify all spam warnings..." message at the bottom
![image](https://github.com/user-attachments/assets/f63ff33f-d235-4b42-901d-4ab11f390576)

**Reply by ryuhangyeong** [reply]
<img width="1208" alt="image" src="https://github.com/user-attachments/assets/97cab9f2-5b62-4baf-b030-cf6f3c5f721c">

<img width="842" alt="image" src="https://github.com/user-attachments/assets/750e42fd-15a0-4501-8bcc-042f1542ecde">

Is this what you want?

**Reply by joshenlim** [reply]
Yup! Are you able to save your email template after making changes?

**Reply by ryuhangyeong** [reply]
Yes, I have confirmed the change!

**Reply by ryuhangyeong** [reply]
Perhaps I was mistaken. Thank you for your support.

**Reply by joshenlim** [reply]
no worries at all! 😄 just feel free to let us know if you're running into any other issues 🙂 🙏
